src/services/TechRecordsService.ts

```diff
@@ -373,6 +419,45 @@ class TechRecordsService {
         }
     return new HTTPResponse(200, this.formatTechRecordItemForResponse(updatedTechRecord.Attributes as ITechRecordWrapper));
   }
+
+  public addProvisionalTechRecord(techRecord: ITechRecordWrapper, msUserDetails: IMsUserDetails) {
+    return this.addNewProvisionalRecord(techRecord, msUserDetails)
+    .then((data: any) => {
+      return this.techRecordsDAO.updateSingle(data)
+        .then((updatedData: any) => {
+          return this.formatTechRecordItemForResponse(updatedData.Attributes);
+        })
+        .catch((error: any) => {
+          throw new HTTPError(error.statusCode, error.message);
+        });
+    })
+    .catch((error: any) => {
+      throw new HTTPError(error.statusCode, error.body);
+    });
+  }
+
+  public addNewProvisionalRecord(techRecordToAdd: ITechRecordWrapper, msUserDetails: any) {
```

Comment:
msUserDetails can use IMsUserDetails. Also we should validate the new record and allow the creation only if it passes validation